dev/io.openliberty.data.internal_fat/test-applications/DataTestApp/src/test/jakarta/data/web/DataTestServlet.java

```diff
@@ -3066,6 +3068,23 @@ public void testCursorPreviousWithUpdates() {
         }
     }
 
+    @Test //TODO
+    @Ignore("Reference issue: https://github.com/OpenLiberty/open-liberty/issues/29475")
+    public void testFetchTypeDefault() {
+        ratings.clear();
+
+        Rating.Reviewer user1 = new Rating.Reviewer("Rex", "TestRecordWithEmbeddables", "rex@openliberty.io");
```

Comment:
It looks like this copied a test case name from a different test. Recommend updating it to avoid causing confusion to anyone debugging issues in the future:
```suggestion
        Rating.Reviewer user1 = new Rating.Reviewer("Rex", "TestFetchTypeDefault", "rex@openliberty.io");
```